web/app/scripts/src/providers/project-config.es.js

```diff
@@ -0,0 +1,22 @@
+const ProjectConfig = function () {
+    let settings = {
+        apiRoot: 'localhost:8000'
+    };
+
+    return {
+        $get: function () {
+            return {
+                getApiRoot: () => settings.apiRoot
```

Comment:
I think it'd be simpler to `return settings`, so you don't have to add a get method for every new property. But I see how that means the data can be changed. I think simpler would be better here: less work to update for changes